lighthouse/tests/validator_client.rs

```diff
@@ -317,6 +317,14 @@ fn missing_unencrypted_http_transport_flag() {
         .with_config(|config| assert_eq!(config.http_api.listen_addr, addr));
 }
 #[test]
+#[should_panic]
+fn msising_http_http_port_flag() {
```

Comment:
Just a small nitpick about the spelling

```suggestion
fn missing_http_http_port_flag() {
```